Resources/Locales/ko/openspades.po

```diff
@@ -1074,10 +1074,10 @@ msgid ""
 "FXAA: Performs antialiasing by smoothing artifacts out as a post-process.|"
 "Off|MSAA 2x|4x|FXAA"
 msgstr ""
-"앤티앨리어싱 :오브젝트의 윤곽에 발생하는 계단 현상을 줄여 줍니다.\n"
+"앤티앨리어싱:오브젝트의 윤곽에 발생하는 계단 현상을 줄여 줍니다.\n"
 "\n"
-"MSAA : 보통의 경우보다 고해상도의 이미지를 생성하여 앤티앨리어싱을 수행합니"
-"다. 가장 멋져 보이지만, 몇몇 설정과는 호환되지 않습니다.\n"
+"MSAA:보통의 경우보다 고해상도의 이미지를 생성하여 앤티앨리어싱을 수행합니다. "
```

Comment:
Can you make spaces around punctuations consistent with the original text, or at least other translated texts?